docs/http/traffic-policy/gallery.mdx

```diff
@@ -72,3 +72,31 @@ This rule applies rate limiting of `30` requests per second to the endpoint `/ap
 		],
 	}}
 />
+
+## User agent filtering
+
+Here we are serving custom content to Microsoft Edge users by checking whether the `User-Agent` header has case insensitive `(?i)` string `edg/` followed by some digits `\d`. You can test this by viewing this [regex101 link](https://regex101.com/r/3NPVub/1).
+
+Additionally, we have to escape the `\d` properly so it can be decoded from YAML / JSON. In YAML when your string is not wrapped in quotes you single-escape: `\\d` when it is wrapped in quotes you must double-escape: `\\\\d` for it to properly decode.
```

Comment:
Question: The first sentence says we need to escape the `\d` properly for YAML and JSON decoding. However, the second sentence starts with "In YAML". Is the text that follows true for YAML only or for JSON as well? If YAML only, we should also add information on what needs to be done for JSON.

Also, the second sentence should be two sentences, maybe something like ...
```
Strings that are not wrapped in quotes are single-escaped: `\\d`. Strings wrapped in quotes are double-escaped: `\\\\d`.
```

Reply:
cc: @nijikokun

Reply:
JSON is always 4